libraries/AP_HAL_ChibiOS/CANFDIface.cpp

```diff
@@ -996,13 +992,10 @@ uint32_t CANIface::getErrorCount() const
 }
 
 #if CH_CFG_USE_EVENTS == TRUE
```

Comment:
Depends on Counting semaphore enabled now. This condition needs to be removed. There might be other places where this is done.

Reply:
counting semaphores are always enabled, condition removed